Formula/dvm.rb

```diff
@@ -15,14 +15,16 @@ class Dvm < Formula
 
   def install
     ENV["GOPATH"] = buildpath
+    ENV["PATH"] = "#{ENV["PATH"]}:#{buildpath}/bin"
```

Comment:
Perhaps worth doing:
```ruby
ENV.append_path "PATH", buildpath/"bin"
```
here if it works, for the sake of looking a bit neater 🙈.